### PR TITLE
How to build a simple Angular SPA from scratch

### PR DESCRIPTION
**Pull request by MahanteshR**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by zolomohan** [comment]
@MahanteshR  Please update the PR title to the Article's title.

**Comment by MahanteshR** [comment]
This fixes #1062

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

**Comment by jumamiller** [comment]
Hi @hectorkambow, I'll review this.

File: articles/building-a-simple-angular-app/index.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
+Angular 11 was released on Nov 11, 2020, by the Angular team at Google.
+Angular's development community has been growing over the past few years.
+If you aren't familiar with Angular, there are plenty of reasons why you should start learning.
+This year's release focuses on improving the developer's experience.
+To check out more about releases, [click here](#https://blog.angular.io/version-11-of-angular-now-available-74721b7952f7).
```
Comment (jumamiller) [inline articles/building-a-simple-angular-app/index.md:5]: Hi @MahanteshR, the link provided is not working as expected, consider removing the '#'.

Comment (jumamiller) [inline articles/building-a-simple-angular-app/index.md:5]: Hi @MahanteshR, the link ain't working as expected. Consider removing the '#' at the start.

File: articles/building-a-simple-angular-app/index.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+The guide is handy for developers who are willing to have a hands-on experience, along with learning.
+
+We are going to build a tiny version of a streaming anime application. On the home page,
+we will display all the anime covers. The user can click on any of the cards to be redirected to the respective anime profile section.
+
+The anime profile contains all the details about that respective anime—
+also, a comment box at the bottom.
+### Getting Started
+
+With Node.js installed, run the following command to install Angular CLI.
```
Comment (jumamiller) [inline articles/building-a-simple-angular-app/index.md:18]: Hi, @MahanteshR, redirect users with no prior Node.js knowledge to a page with installation instructions.

File: articles/building-a-simple-angular-app/index.md (hunk 1 above)
Comment (jumamiller) [inline articles/building-a-simple-angular-app/index.md:5]: Hi @MahanteshR, the link ain't working as expected. Consider removing the '#' at the start.

**Review by jumamiller — CHANGES_REQUESTED** [review]
@MahanteshR, job well done! only a single suggestion made. Ping me when you're done. Tech reviews up next.

**Comment by jumamiller** [comment]
@hectorkambow, plagiarism check required.

File: articles/building-a-simple-angular-app/index.md (hunk 1 above)
Comment (MahanteshR) [reply articles/building-a-simple-angular-app/index.md]: Thanks for notifying @jumamiller. The link is working properly now.

**Comment by jumamiller** [comment]
@MahanteshR, tech review in progress.

File: articles/building-a-simple-angular-app/index.md [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+```typescript
+  // anime-card.component.ts
+
+    export class AnimeComponent{
+      @Input()
+      anime: AnimeInterface;
+
+      @Input()
+      animeId: number;
+    }
```
Comment (jumamiller) [inline articles/building-a-simple-angular-app/index.md:171]: The component name 'AnimeComponent' throws an error, kindly modify it to `AnimeCardComponent`(line 165)

**Review by jumamiller — CHANGES_REQUESTED** [review]
@MahanteshR revise the component error. Ping me when done.

File: articles/building-a-simple-angular-app/index.md (hunk 3 above)
Comment (MahanteshR) [reply articles/building-a-simple-angular-app/index.md]: It has been modified and I had missed a file, added that file too.

**Comment by jumamiller** [comment]
@hectorkambow, this tutorial is ready for final review.

**Pull request merged**